plaidml/core/core.h

```diff
@@ -40,6 +40,16 @@ void call_void(F fn, Args... args) {
   }
 }
 
+template <typename F, typename... Args>
+double call_double(F fn, Args... args) {
+  plaidml_error err;
+  double ret = fn(&err, args...);
+  if (err.code) {
+    throw std::runtime_error(str(err.msg));
+  }
+  return ret;
+}
+
```

Comment:
No need for this, there's a templated version already.

Reply:
Done.